apps/web/src/services/dashboards/widgets/_helpers/widget-schema-helper.ts

```diff
@@ -45,6 +47,13 @@ export const getWidgetFilterOptionsSchema = (...filterKeys: WidgetFilterKey[]):
 
 export const getWidgetFilterSchemaPropertyNames = (...keys: WidgetFilterKey[]): WidgetFiltersSchemaProperty[] => keys.map((key) => `filters.${key}` as WidgetFiltersSchemaProperty);
 export const getWidgetFilterSchemaPropertyName = (key: WidgetFilterKey): WidgetFiltersSchemaProperty => `filters.${key}`;
+export const getWidgetOptionsSchemaPropertyName = (property: WidgetOptionsSchemaProperty): string => { // 'filters.project_group' -> 'Project Group'
+    if (property.startsWith('filters.')) {
+        return REFERENCE_TYPE_INFO[property.replace('filters.', '')].name;
+    }
+    if (property === 'group_by') return 'Group By';
+    return property;
+};
```

Comment:
Suggestion: It would be nice if annotation improved a little. That annotation seems to explain only part of it such as 'filters.' case. It's just suggestion.

Reply:
Thanks! I changed it to
```typescript
/** @function
 * @name getWidgetOptionsSchemaPropertyName
 * @example
    * getWidgetOptionsSchemaPropertyName('filters.project_group') => 'Project Group'
    * getWidgetOptionsSchemaPropertyName('group_by') => 'Group By'
 */
```